### ISSUE TITLE
Build instructions are missing

### ISSUE THREAD

**Issue by jorants** [donovansolms/ut4-launcher#1]
I never use go so it took me a while to figure out how to build this (getting deps with "go get").
Maybe a few lines on how to build could help?

**Comment by donovansolms** [comment]
Yes, you are correct. I should have included the build instructions by now - apologies.

Do note that this doesn't actually work yet, I haven't touched it in a year, doubt that I'll get back to this, hopefully Epic will release a launcher.

**Comment by jorants** [comment]
Yes, i noticed that after I got it to compile :P
Maybe an easier solution would be to note the state of the project a bit clearer in the readme?
